backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -658,8 +658,12 @@ public Project SetStatus(Project project, Workflow workflow)
             var toStatus = this.Context.ProjectStatus
                 .Include(s => s.Tasks)
                 .FirstOrDefault(s => s.Id == project.StatusId);
-            if (!fromStatus.ToStatus.Any(s => s.ToStatusId == project.StatusId)) throw new InvalidOperationException($"Invalid project status transitions from '{fromStatus.Name}' to '{toStatus?.Name}'.");
-
+            // TODO: ignore status transitions for ACCESS-DISPOSAL steps for now.
```

Comment:
You shouldn't need to do this, just add additional valid transitions to the sql script.